packages/react-instantsearch-dom/src/hooks/useAnswers.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState, useEffect, useMemo, useContext } from 'react';
+import { instantSearchContext } from 'react-instantsearch-core';
+import { createConcurrentSafePromise } from '../lib/createConcurrentSafePromise';
+import { debounce } from '../lib/debounce';
+
+function hasReactHooks() {
+  // >= 16.8.0
+  const [major, minor] = React.version.split('.').map(Number);
+  return major >= 17 || (major === 16 && minor >= 8);
+}
+
+export default function useAnswers({
+  searchClient,
```

Comment:
this is accessible from the InstantSearch context I think (otherwise should get exposed that way)

Reply:
I just checked and the context doesn't have it. I'll continue that part in a subsequent PR, if you think it's okay.